ext/ftp/tests/server.inc

```diff
@@ -19,7 +19,7 @@ if ($pid) {
     function dump_and_exit($buf)
     {
         var_dump($buf);
-        exit;
+        exit();
```

Comment:
There are still a couple of these, can we revert them?

Reply:
Done